cmd/find-junit5-prs/main.go

```
@@ -0,0 +1,537 @@
+	}
+
+	// Parse start date
+	startDateTime, err := time.Parse("2006-01-02", *startDate)
+	if err != nil {
+		fmt.Printf("Error parsing start date: %v\n", err)
+		os.Exit(1)
+	}
+	fmt.Printf("Searching for PRs created on or after: %s\n", startDateTime.Format("2006-01-02"))
+
+	// Create output directory if it doesn't exist
+	os.MkdirAll(*outputDir, 0755)
```

golangci-lint (1.64.8)

Error return value of `os.MkdirAll` is not checked

(errcheck)